shared/src/hover/HoverOverlay.tsx

```diff
@@ -167,7 +167,13 @@ export class HoverOverlay<A extends string> extends React.PureComponent<HoverOve
                               visibility: 'hidden',
                           }
                 }
-                className={classNames('hover-overlay', className)}
+                className={classNames(
+                    'hover-overlay',
+                    className,
+                    // Ensure theme-light is applied here, otherwise the browser
+                    // extension will always render with a dark theme background.
+                    this.props.isLightTheme ? 'theme-light' : ''
```

Comment:
```suggestion
                    {'theme-light': this.props.isLightTheme}
```

You can use an object to express this in `classNames` rather than using a ternary: https://github.com/JedWatson/classnames#usage-with-reactjs